sdk/eventhub/event-hubs/src/index.ts

```diff
@@ -34,5 +33,5 @@ export { EventDataBatch, TryAddOptions } from "./eventDataBatch";
 export { Checkpoint } from "./partitionProcessor";
 export { CheckpointStore, PartitionOwnership } from "./eventProcessor";
 export { CloseReason } from "./models/public";
-export { MessagingError, RetryOptions, TokenCredential } from "@azure/core-amqp";
+export { MessagingError, RetryOptions, TokenCredential, WebSocketOptions } from "@azure/core-amqp";
```

Comment:
Can we update the version of `@azure/core-amqp` that event hubs depends on to match what's in master? That'll be important given `@azure/core-amqp` 1.0.1 and lower wasn't exporting WebSocketOptions before.

Reply:
Updated the version in event-hubs and service-bus to core-amqp `^1.0.2`, added changelog for core-amqp.